src/main/java/com/stripe/param/FileCreateParams.java

```diff
@@ -0,0 +1,253 @@
+        return new FileLinkData(this.create, this.expiresAt, this.metadata);
+      }
+
+      /** Set this to `true` to create a file link for the newly created file. */
+      public Builder setCreate(Boolean create) {
+        this.create = create;
+        return this;
+      }
+
+      /** A future timestamp after which the link will no longer be usable. */
+      public Builder setExpiresAt(Long expiresAt) {
+        this.expiresAt = expiresAt;
+        return this;
+      }
+
+      /**
+       * Add a key/value pair to `metadata` map. A map is initialized for the first `put/putAll`
+       * call, and subsequent calls add additional key/value pairs to the original map. See {@link
+       * FileLinkData#metadata} for the field documentation.
+       */
+      public Builder putMetadata(String key, String value) {
+        if (this.metadata == null) {
+          this.metadata = new HashMap<>();
+        }
+        this.metadata.put(key, value);
+        return this;
+      }
+
+      /**
+       * Add all map key/value pairs to `metadata` map. A map is initialized for the first
+       * `put/putAll` call, and subsequent calls add additional key/value pairs to the original map.
+       * See {@link FileLinkData#metadata} for the field documentation.
+       */
+      public Builder putAllMetadata(Map<String, String> map) {
+        if (this.metadata == null) {
+          this.metadata = new HashMap<>();
+        }
+        this.metadata.putAll(map);
+        return this;
+      }
+    }
+  }
+
+  public enum Purpose implements ApiRequestParams.EnumParam {
+    @SerializedName("business_icon")
+    BUSINESS_ICON("business_icon"),
+
+    @SerializedName("business_logo")
+    BUSINESS_LOGO("business_logo"),
+
+    @SerializedName("customer_signature")
+    CUSTOMER_SIGNATURE("customer_signature"),
+
+    @SerializedName("dispute_evidence")
+    DISPUTE_EVIDENCE("dispute_evidence"),
+
+    @SerializedName("identity_document")
+    IDENTITY_DOCUMENT("identity_document"),
+
+    @SerializedName("pci_document")
+    PCI_DOCUMENT("pci_document"),
+
+    @SerializedName("tax_document_user_upload")
+    TAX_DOCUMENT_USER_UPLOAD("tax_document_user_upload");
+
+    @Getter(onMethod = @__({@Override}))
+    private final String value;
+
+    Purpose(String value) {
+      this.value = value;
+    }
+  }
+
+  /**
+   * Returns untyped parameters for file creation. Map value for {@code "file"} is the same
+   * instance of value set in {@link Builder#setFile(File)} or
+   * {@link Builder#setFile(java.io.FileInputStream)}; the file is not transformed or serialized
+   * at this level.
+   * @return Untyped parameters containing file object set in the builder.
+   */
+  @Override
+  public Map<String, Object> toMap() {
+    Object fileObject = this.file;
+    Map<String, Object> untypedParamWithPrimitiveTypes = super.toMap();
+    untypedParamWithPrimitiveTypes.put("file", fileObject);
+    return untypedParamWithPrimitiveTypes;
```

Comment:
Because `toMap` needs to contain the same file instance set in the builder

Reply:
👍